spec/test-outputs/www-integration.out.vcl

```diff
@@ -148,9 +148,15 @@ sub vcl_recv {
      error 801 "Force SSL";
   }
 
+  # Reject any methods other than the ones we support.
+  # No need to include FASTLYPURGE here as that's handled above.
+  if (req.request != "HEAD" && req.request != "GET" && req.request != "POST") {
+    error 804 "Invalid Method";
+  {
```

Comment:
This looks odd, should it be `}`?

Would it be better to change this to a [405 Method Not Allowed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405.)? I'm thinking that in a couple of years somebody will use a PUT or PATCH in a frontend app and not understand why it 404s on production - a 405 will be more revealing.

Reply:
Yes, it should be `}`, that's a typo.

Technically `405` is indeed the right response type. In the past I've had folks doing security testing saying that it's better to be more ambiguous about the return value (e.g. "`404` tells me that there's nothing there" vs. "`405` tells me there might be something here and I should keep digging.") But personally I'm happy to make it a "proper" `405`.